### ISSUE TITLE
Funannotate compare stuck at the Proteinortho step for 2 days. The command is at sleep.

### ISSUE THREAD

**Issue by akshayoo** [nextgenusfs/funannotate#1076]
funannotate compare -i Cross_sp_Compare_GBK/*.gbk -o Cross_compare_results --cpus 24
-------------------------------------------------------
[Oct 22 10:15 AM]: OS: Ubuntu 20.04, 24 cores, ~ 231 GB RAM. Python: 3.8.19
[Oct 22 10:15 AM]: Running 1.8.17
[Oct 22 10:15 AM]: Now parsing 11 genomes
[Oct 22 10:15 AM]: working on Fusarium boothii
[Oct 22 10:16 AM]: working on Fusarium equiseti
[Oct 22 10:16 AM]: working on Fusarium hainanense
[Oct 22 10:16 AM]: working on Fusarium humuli
[Oct 22 10:17 AM]: working on Fusarium irregulare
[Oct 22 10:17 AM]: working on Fusarium kyushuense
[Oct 22 10:17 AM]: working on Fusarium langsethiae
[Oct 22 10:17 AM]: working on Fusarium luffae
[Oct 22 10:18 AM]: working on Fusarium pseudograminearum
[Oct 22 10:18 AM]: working on Fusarium equiseti SF5
[Oct 22 10:18 AM]: working on Fusarium vanenatum
[Oct 22 10:19 AM]: No secondary metabolite annotations found
[Oct 22 10:19 AM]: Summarizing PFAM domain results
/home/user/.local/lib/python3.8/site-packages/sklearn/manifold/_mds.py:298: FutureWarning: The default value of `normalized_stress` will change to `'auto'` in version 1.4. To suppress this warning, manually set the value of `normalized_stress`.
  warnings.warn(
[Oct 22 10:19 AM]: Summarizing InterProScan results
[Oct 22 10:19 AM]: Loading InterPro descriptions
/home/user/.local/lib/python3.8/site-packages/sklearn/manifold/_mds.py:298: FutureWarning: The default value of `normalized_stress` will change to `'auto'` in version 1.4. To suppress this warning, manually set the value of `normalized_stress`.
  warnings.warn(
[Oct 22 10:19 AM]: Summarizing MEROPS protease results
[Oct 22 10:19 AM]: found 13/102 MEROPS familes with stdev >= 1.000000
[Oct 22 10:19 AM]: Summarizing CAZyme results
[Oct 22 10:19 AM]: found 24/157 CAZy familes with stdev >= 1.000000
[Oct 22 10:19 AM]: Summarizing COG results
[Oct 22 10:19 AM]: No SignalP annotations found
[Oct 22 10:19 AM]: Summarizing fungal transcription factors
[Oct 22 10:19 AM]: Running GO enrichment for each genome
  WARNING: skipping Fusarium_vanenatum.txt as no GO terms
  WARNING: skipping Fusarium_irregulare.txt as no GO terms
  WARNING: skipping Fusarium_humuli.txt as no GO terms
  WARNING: skipping Fusarium_hainanense.txt as no GO terms
  WARNING: skipping Fusarium_equiseti.txt as no GO terms
  WARNING: skipping Fusarium_pseudograminearum.txt as no GO terms
  WARNING: skipping Fusarium_boothii.txt as no GO terms
  WARNING: skipping Fusarium_kyushuense.txt as no GO terms
  WARNING: skipping Fusarium_luffae.txt as no GO terms
  WARNING: skipping Fusarium_langsethiae.txt as no GO terms
[Oct 22 10:19 AM]: Running orthologous clustering tool, ProteinOrtho.  This may take awhile...

**Comment by akshayoo** [comment]
funannotate check --show-versions
-------------------------------------------------------
Checking dependencies for 1.8.17
-------------------------------------------------------
You are running Python v 3.8.19. Now checking python packages...
biopython: 1.79
goatools: 1.4.12
matplotlib: 3.7.5
natsort: 8.4.0
numpy: 1.24.4
pandas: 2.0.3
psutil: 6.0.0
requests: 2.32.3
scikit-learn: 1.3.2
scipy: 1.10.1
seaborn: 0.13.2
All 11 python packages installed


You are running Perl v b'5.032001'. Now checking perl modules...
Carp: 1.50
Clone: 0.46
DBD::SQLite: 1.72
DBD::mysql: 4.050
DBI: 1.643
DB_File: 1.858
Data::Dumper: 2.183
File::Basename: 2.85
File::Which: 1.24
Getopt::Long: 2.58
Hash::Merge: 0.302
JSON: 4.10
LWP::UserAgent: 6.67
Logger::Simple: 2.0
POSIX: 1.94
Parallel::ForkManager: 2.03
Pod::Usage: 1.69
Scalar::Util::Numeric: 0.40
Storable: 3.15
Text::Soundex: 3.05
Thread::Queue: 3.14
Tie::File: 1.06
URI::Escape: 5.17
YAML: 1.30
local::lib: 2.000029
threads: 2.25
threads::shared: 1.61
All 27 Perl modules installed


Checking Environmental Variables...
$FUNANNOTATE_DB=/home/user/funannotate_db/
$PASAHOME=/home/user/anaconda3/envs/funannotate/opt/pasa-2.5.3
$TRINITYHOME=/home/user/anaconda3/envs/funannotate/opt/trinity-2.1.1
$EVM_HOME=/home/user/anaconda3/envs/funannotate/opt/evidencemodeler-2.1.0
$AUGUSTUS_CONFIG_PATH=/home/user/anaconda3/envs/augustus/config/
	ERROR: GENEMARK_PATH not set. export GENEMARK_PATH=/path/to/dir
-------------------------------------------------------
Checking external dependencies...
PASA: 2.5.3
CodingQuarry: 2.0
Trinity: 2.15.2
augustus: 3.5.0
bamtools: bamtools 2.5.2
bedtools: bedtools v2.31.1
blat: BLAT v39x1
diamond: 2.1.8
emapper.py: 2.1.12-3-g3666cb0
ete3: 3.1.3
exonerate: exonerate 2.4.0
fasta: 36.3.8g
glimmerhmm: 3.0.4
gmap: 2024-10-10
hisat2: 2.2.1
hmmscan: HMMER 3.4 (Aug 2023)
hmmsearch: HMMER 3.4 (Aug 2023)
java: 22.0.1-internal
kallisto: 0.46.1
mafft: v7.526 (2024/Apr/26)
makeblastdb: makeblastdb 2.16.0+
minimap2: 2.28-r1209
pigz: 2.8
proteinortho: 6.3.2
pslCDnaFilter: no way to determine
salmon: salmon 1.10.3
samtools: samtools 1.21
snap: 2006-07-28
stringtie: 2.2.3
tRNAscan-SE: 2.0.12 (Nov 2022)
tantan: tantan 50
tbl2asn: 25.8
tblastn: tblastn 2.16.0+
trimal: trimAl v1.5.rev0 build[2024-05-27]
trimmomatic: 0.39
	ERROR: gmes_petap.pl not installed
	ERROR: signalp not installed

**Comment by hyphaltip** [comment]
unclear the comparative tool is that robust - I would look at other strategies for orthology like OrthoFinder.
